client/src/components/user/UserRouter.js

```diff
@@ -44,7 +44,7 @@ const UserRouter = () => {
           });
     };
     checkLoggedIn();
-  });
```

Comment:
I diced into docs and figured that a dependency array is required to avoid re-render loops. 
Adding an empty array resulted in throwing a warning saying that  `login` is not included.
Hence, I included warning.